### PR TITLE
hide vie link on posts in admin panel, if posts there are has no single

### PR DESCRIPTION
**Pull request by aprokopenko**

### CODE REVIEW

File: app/Post_Type/Hero.php [hunk 1]
```diff
@@ -31,7 +31,9 @@ public function init() {
 
 		$this->has_single    = false;
 		$this->is_searchable = false;
-		$this->redirect      = home_url();
+		// redirect is commented - to hide view link on the admin panel
+		// do this if has_single == false
+		//$this->redirect      = home_url();
```
Comment (aprokopenko) [inline app/Post_Type/Hero.php]: please investigate first why this redirect is set here!

I dont' think you were right commenting it.

**Review by aprokopenko — COMMENTED** [review]
Please answer questions

File: app/Post_Type/Hero.php (hunk 1 above)
Comment (mzhirov) [reply app/Post_Type/Hero.php]: A redirect is needed so that you can not make the transition to a post of a closed content type

**Comment by aprokopenko** [comment]
nothing to update

**Pull request closed without merging**